### PR TITLE
[LIBS-258] Add support for PCSS, autofix code at git hooks

### PR DESCRIPTION
**Pull request by cesargdm**
We're now supporting PCSS and removed the linting from react-scripts, giving this task to the new package https://github.com/credijusto/eslint-config-cj-app, actually, I need also to publish a stylelint package..., and the set of husky hooks ~, I think the best is to set them at the package-level~ , working on setting right the custom husky hooks to auto-fix linter problems

### CODE REVIEW

**Comment by jmz7v** [comment]
Forgot to read the description... my bad.

**Comment by cesargdm** [comment]
> There's a way to lint PostCSS, can you take a look at it to see if it'd be compatible with this new version?
> https://stylelint.io/user-guide/postcss-plugin/

Sure, working on that...

**Comment by aegre** [comment]
Does this remove sass compatibility?

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -63,44 +63,35 @@
...
     "resolve": "1.10.0",
     "sass-loader": "7.1.0",
     "semver": "6.0.0",
     "style-loader": "0.23.1",
-    "stylelint": "9.10.1",
-    "stylelint-config-rational-order": "0.1.2",
-    "stylelint-config-standard": "18.2.0",
-    "stylelint-declaration-block-no-ignored-properties": "2.1.0",
-    "stylelint-order": "3.0.0",
-    "stylelint-scss": "3.5.3",
```
Comment (dyegolara) [inline packages/react-scripts/package.json]: all of these are required on our current version of the scripts, when we upgrade to `js-scripts` we can even not use this package, in the meanwhile, please keep them

**Comment by jmz7v** [comment]
@aegre I think it does, should we wait for loany?

**Comment by aegre** [comment]
@jmz7v se can try but take in mind that loany will be excluded in all the new things, maybe another redactor will be needed just like when se add cra to loany

**Comment by cesargdm** [comment]
> Does this remove sass compatibility?

No, it does not, just we're removing the linting and pre-commit responsibilities from this package.

**Comment by jmz7v** [comment]
@cesargdm The only project that does not rely on it is credijusto.com, all other projects still require it (crap, engineering, loany, portal).

**Comment by cesargdm** [comment]
@jmz7v Sure, It would only require to manually add node-sass to those projects after the upgrade

**Pull request merged**